src/lib.rs

```diff
@@ -148,6 +155,9 @@ pub struct ConversionOptions {
     ///
     /// _Default:_ `true`.
     pub embed_text: bool,
+
+    /// Whether to enforce PDF/A rules.
+    pub pdfa: bool,
```

Comment:
I would add a comment that this doesn't refer to making the whole file PDF/A compliant, though. From what I remember, there are other things missing, like adding certain metadata, which presumably will be implemented on the Typst side. 

But if you just use svg2pdf by itself, the resulting PDF won't be PDF/A compliant.

Reply:
True! I missed that.